source/appModules/securecrt.py

```diff
@@ -2,11 +2,10 @@
 #A part of NonVisual Desktop Access (NVDA)
 #This file is covered by the GNU General Public License.
 #See the file COPYING for more details.
-#Copyright (C) 2010-2012 NV Access Limited
+#Copyright (C) 2010-2016 NV Access Limited
 
 """App module for automatic reading of the terminal in all versions of SecureCRT
-"""
-
+Uses regular expressions to support the ever-changing window title for different incremental SecureCRT Versions."""

```

Comment:
While this might be good as a comment, it's an implementation detail and I don't think it needs to be part of the module docstring.